libs/packages/components/src/lib/menu/menu.module.ts

```diff
@@ -6,9 +6,10 @@ import { SdsMenuItemComponent } from './menu-item.component';
 import { SdsMenuTriggerForDirective } from './menu-trigger.directive';
 import { SdsMenuHeaderComponent } from './menu-header.component';
 import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
+import { SdsIconModule } from '../icon/icon.module';
 
 @NgModule({
-  imports: [CommonModule, OverlayModule, FontAwesomeModule],
+  imports: [CommonModule, OverlayModule, FontAwesomeModule, SdsIconModule],
```

Comment:
Are we still using fa-icon in menu components? If not, then I think we can remove the dependency on FontAwesomeModule here.